File: net.tenteCsharp/src_code/heaterManagement/IHeating.java
package heaterManagement;
import heaterManagement.HeaterManagement.*;

public interface IHeating{
	
	public void setPower(String heaterId,int amount);	
	public void setMode(String heaterId,HeatingModes mode);	
	public void heatingSwitch(String heaterId,boolean on);	
	public void setTemperature(String heaterId,float temp);

}

File: trunk/Elio/pseudoCodeGeneratorElio/src-gen/heaterManagement/IHeating.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface IHeating
	{
		void setPower(String heaterId, int amount);
		void setMode(String heaterId, HeatingModes mode);
		void heatingSwitch(String heaterId, Boolean on);
		void setTemperature(String heaterId, float temp);

	}
}
